packages/babel-generator/src/generators/typescript.ts

```diff
@@ -347,7 +351,11 @@ export function TSIntersectionType(this: Printer, node: t.TSIntersectionType) {
   this.tsPrintUnionOrIntersectionType(node, "&");
 }
 
-export function tsPrintUnionOrIntersectionType(this: Printer, node: any, sep) {
+export function tsPrintUnionOrIntersectionType(
```

Comment:
We don't need to export this, or `tsPrintBraced`

Reply:
Yes, and there are many other internal printer methods, too. I will take care of them in another PR.